es/flask-web-apps-workshop/index.html

```diff
@@ -10,8 +10,9 @@
     <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/reveal.js@4.5.0/dist/theme/simple.css" id="theme">
     <link rel="stylesheet"
         href="https://fonts.googleapis.com/css?family=Roboto+Mono|Roboto:300,400,500|Work+Sans:400,700">
-    <link rel="stylesheet" href="../../../static/a11y-light.min.css">
-    <link rel="stylesheet" href="../../../static/slides.css">
+    <link rel="stylesheet" href="https://raw.githubusercontent.com/pamelafox/python-web-apps/main/static/a11y-light.min.css">
```

Comment:
That will make it harder for you to test changes locally to any of those, but at this point, they should be fairly constant, so am okay with that if you are.

Reply:
Actually would it be better to link to the github.io URL instead? Content headers might be better than for raw.githubusercontent.com